lbz/websocket/request.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from multidict import CIMultiDict
+
+from lbz._request import _Request
+from lbz.authentication import User
+from lbz.misc import get_logger
+from lbz.websocket.enums import ActionType
+
+logger = get_logger(__name__)
```

Comment:
It is not used here as well:
```suggestion
from lbz.websocket.enums import ActionType
```